k8s.gcr.io/images/k8s-staging-e2e-test-images/images.yaml

```diff
@@ -59,6 +59,7 @@
     "sha256:dc81c9e400528c35e5b58d5208e7aece2b7452985009a44a341f42bebaabf07b": ["1.14-alpine"]
     "sha256:05ac4b4aef14d4a594afc8e6a63076197debfee2a7b8c191f65c7d653beee0e0": ["1.15-alpine"]
     "sha256:503b7abb89e57383eba61cc8a9cb0b495ea575c516108f7d972a6ff6e1ab3c9b": ["1.14-1"]
+    "sha256:503b7abb89e57383eba61cc8a9cb0b495ea575c516108f7d972a6ff6e1ab3c9b": ["1.14-1-copy"]
```

Comment:
Adding an identical key breaks the unmarshaling logic. Instead you should just reuse the existing key in the line above, but with the new tag added as a 2nd element, like this:

```
"sha256:503b7abb89e57383eba61cc8a9cb0b495ea575c516108f7d972a6ff6e1ab3c9b": ["1.14-1", "1.14-1-copy"]
```

Reply:
Ah, I see. Ty!